devicetypes/smartthings/smartsense-temp-humidity-sensor.src/smartsense-temp-humidity-sensor.groovy

```diff
@@ -28,9 +28,7 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0001,0003,0020,0402,0B05,FC45", outClusters: "0019,0003", manufacturer: "CentraLite", model: "3310-S", deviceJoinName: "SmartSense Temp & Humidity Sensor"
 		fingerprint profileId: "0104", inClusters: "0001,0003,0020,0402,0B05,FC45", outClusters: "0019,0003", manufacturer: "CentraLite", model: "3310-G", deviceJoinName: "Centralite Temp & Humidity Sensor"
 		fingerprint profileId: "0104", inClusters: "0001,0003,0020,0402,0B05,FC45", outClusters: "0019,0003", manufacturer: "CentraLite", model: "3310", deviceJoinName: "Temp & Humidity Sensor"
-		fingerprint profileId: "0104", deviceId: "0302", inClusters: "0000,0001,0003,0402", manufacturer: "Heiman", model: "b467083cfc864f5e826459e5d8ea6079", deviceJoinName: "Orvibo Temperature & Humidity Sensor"
```

Comment:
Why are these fingerprints removed?

Reply:
the orvibo temp&humidity sensor device production has stopped 2 years ago, and there is also no plan to publish the catalog in public in CN region, the orvibo sensor is just for practice of DTH development in WWST Nanjing office in year 2018. so I just remove orvibo sensor fingerprint.

Reply:
Even if production has stopped we like to keep the fingerprints around just in case someone buys one secondhand.

Reply:
got it